README.md

```diff
@@ -27,66 +27,69 @@ This client supports the following Google Cloud Platform services:
 > Note: This client is a work-in-progress, and may occasionally
 > make backwards-incompatible changes.
 
+## Where did `gcloud-java` go?
+
+`gcloud-java` lives on under a new name, `google-cloud`.
```

Comment:
Or not so simple: The repo is `google-cloud-java` and the artifacts are `google-cloud`.